product_hunt_utils.py

```diff
@@ -69,6 +68,173 @@ def get_commentor_info_for_product_id(product_id: int)-> list:
+}
+
+    first_response = requests.post('https://www.producthunt.com/frontend/graphql', cookies=cookies, headers=headers, json=json_data)
+
+    first_response = first_response.json()
+
+    first_followings = first_response["data"]["profile"]["following"]["edges"]
+    end_cursor = first_response["data"]["profile"]["following"]["pageInfo"]["endCursor"]
+    next_page = first_response["data"]["profile"]["following"]["pageInfo"]["hasNextPage"]
+
+    for nodes in first_followings:
+            id_lst.append(nodes["node"]["id"])
+
+    while(next_page == True):
+        json_data = {
+        'operationName': 'ProfileFollowingQuery',
+        'variables': {
+            'username': 'alicea_hughes',
+            'cursor': end_cursor,
+            'query': '',
+        },
+        'query': 'query ProfileFollowingQuery($username:String!$cursor:String){profile:user(username:$username){id isTrashed followingsCount following(first:20 after:$cursor){edges{node{id ...UserItemFragment __typename}__typename}pageInfo{endCursor hasNextPage __typename}__typename}...MetaTags __typename}}fragment MetaTags on SEOInterface{id meta{canonicalUrl creator description image mobileAppUrl oembedUrl robots title type author authorUrl __typename}__typename}fragment UserItemFragment on User{id name headline username followersCount ...UserImage ...UserFollowButtonFragment __typename}fragment UserFollowButtonFragment on User{id followersCount isFollowed __typename}fragment UserImage on User{id name username avatarUrl __typename}',
+    }
+
+
+        response = requests.post('https://www.producthunt.com/frontend/graphql', cookies=cookies, headers=headers, json=json_data)
+
+        response = response.json()
+
+        followings = response["data"]["profile"]["following"]["edges"]
+
+        end_cursor = response["data"]["profile"]["following"]["pageInfo"]["endCursor"]
+
+        next_page = response["data"]["profile"]["following"]["pageInfo"]["hasNextPage"]
+
+        for nodes in followings:
+            id_lst.append(nodes["node"]["id"])
+    
+    df = pd.DataFrame(id_lst, columns = ["person id"])
+
+
+    df.to_csv('person_followings_ids.csv', index = False)
+
+    df = pd.read_csv('person_followings_ids.csv')
+
+    follow_count = 0
+
+    while(True):
+
+        for i,r in df.iterrows():
+            time.sleep(5)
+            if(follow_a_person(int(r["person id"]))):
+                    follow_count += 1
+                    df = df.drop(i)
+            else:
+                    break
+        if(df.empty):
+              print("All followings are followed")
+              os.remove('person_followings_ids.csv')
+              return True

```

Comment:
Have you tested this part? When I run this function for the second time will it read the csv and update it correctly?